tests/tests_common_test.go

```diff
@@ -111,6 +111,53 @@ func expectRecreateAfterDelete(res *testResource) {
 	Expect(err).ToNot(HaveOccurred())
 }
 
+func sspOperatorReconcilingProperly() (sum int) {
+	operatorPods, operatorMetricsPort := operatorPodsWithMetricsPort()
+	for _, sspOperator := range operatorPods {
+		sum += intMetricValue("ssp_operator_reconciling_properly", operatorMetricsPort, &sspOperator)
+	}
+	return
+}
+
+func totalRestoredTemplatesCount() (sum int) {
+	operatorPods, operatorMetricsPort := operatorPodsWithMetricsPort()
+	for _, sspOperator := range operatorPods {
+		sum += intMetricValue("total_restored_common_templates", operatorMetricsPort, &sspOperator)
+	}
+	return
+}
+
+func operatorPodsWithMetricsPort() ([]core.Pod, uint16) {
+	pods := &core.PodList{}
+	err := apiClient.List(ctx, pods, client.MatchingLabels{"control-plane": "ssp-operator"})
+	Expect(err).ToNot(HaveOccurred())
+	Expect(pods.Items).ToNot(BeEmpty())
+	operatorMetricsPort, err := metricsPort(pods.Items)
+	Expect(err).ToNot(HaveOccurred())
+	return pods.Items, operatorMetricsPort
+}
+
+func metricsPort(pods []core.Pod) (uint16, error) {
```

Comment:
This function is also created in first commit and modified in the second.

Reply:
squashing